modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFClaim.macro

```diff
@@ -93,6 +93,13 @@ definition {
 		return ${orderID};
 	}
 
+	@summary = "Go to MDF Claim"
+	macro gotoMDFClaim {
+		Click(
+			key_index = ${goClaim},
+			locator1 = "PRMMDFClaim#CLAIM_ID_INDEX");
+	}
```

Comment:
Hi @marinaasant0s, you can't use this path here, because I want you to click in the Claim ID, instead of click in the first Claim